src/Microsoft.TemplateSearch.Common/Microsoft.TemplateSearch.Common.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj" />
+    <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Edge\Microsoft.TemplateEngine.Edge.csproj" />
```

Comment:
no-no, please. 
Let's pass the list of templates instead of manager. Common should not depend on Edge.